### PR TITLE
Release eui 2100 master

### PR DESCRIPTION
**Pull request by udaydenduluri33**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: api/inviteUser/inviteUserUtil.spec.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { expect } from 'chai'
+import { getInviteUserUrl } from './inviteUserUtil'
+
+describe('inviteuser util', () => {
+    it('should getInviteUserUrl', () => {
+        let url = getInviteUserUrl('http://base')
+        expect(url).to.equal('http://base/refdata/external/v1/organisations/users/')
+
+        url = getInviteUserUrl('http://base')
+        expect(url).to.equal('http://base/refdata/external/v1/organisations/users/')
```
Comment (sreekanthpuligadda) [inline api/inviteUser/inviteUserUtil.spec.ts:10]: duplicate code from line 6 and 7

File: src/app/components/service-down/service-down.component.ts [hunk 2]
```diff
@@ -1,10 +1,33 @@
...
+        };
+        this.store.pipe(select(fromAppStore.getCurrentError))
+        .subscribe(error => {
+            if (error) {
+                this.currentError = error;
+            }
+        });
+    }
+
+    public showErrorLinkWithNewTab(newTab?: boolean): string {
```
Comment (sreekanthpuligadda) [inline src/app/components/service-down/service-down.component.ts:30]: This method is not used anywhere.

Comment (udaydenduluri33) [reply src/app/components/service-down/service-down.component.ts:30]: Good point. missed it in merge. Added it now

File: api/inviteUser/inviteUserUtil.spec.ts (hunk 1 above)
Comment (udaydenduluri33) [reply api/inviteUser/inviteUserUtil.spec.ts:10]: Would not worry about it in spec files. But, good point

**Pull request merged**